Makefile

```diff
@@ -147,15 +152,46 @@ MAKE_SURE_BIN_EXIST := $(shell mkdir -p dist .go-pkg-cache $(NODE_CONTAINER_BIN_
 NODE_CONTAINER_FILES=$(shell find ./filesystem -type f)
 LOCAL_USER_ID?=$(shell id -u $$USER)
 
-LDFLAGS=-ldflags "-X github.com/projectcalico/node/pkg/startup.VERSION=$(CALICO_GIT_VER) \
-                  -X github.com/projectcalico/node/vendor/github.com/projectcalico/felix/buildinfo.GitVersion=$(CALICO_GIT_VER) \
-                  -X github.com/projectcalico/node/vendor/github.com/projectcalico/felix/buildinfo.GitRevision=$(shell git rev-parse HEAD || echo '<unknown>')"
+# Calculate a timestamp for any build artefacts.
+DATE:=$(shell date -u +'%FT%T%z')
+
+# Figure out version information.  To support builds from release tarballs, we default to
+# <unknown> if this isn't a git checkout.
+GIT_COMMIT:=$(shell git rev-parse HEAD || echo '<unknown>')
+GIT_DESCRIPTION:=$(shell git describe --tags --dirty --always || echo '<unknown>')
+ifeq ($(LOCAL_BUILD),true)
+        GIT_DESCRIPTION = $(shell git describe --tags --dirty --always || echo '<unknown>')-dev-build
+endif
+
+LDFLAGS=-ldflags "\
+	-X $(PACKAGE_NAME)/buildinfo.Version=$(CALICO_GIT_VER) \
```

Comment:
The actual `calico-node -v` command uses `github.com/projectcalico/node/pkg/startup.VERSION` so you need to either set that here and remove `buildinfo.Version` or remove `startup.VERSION` and replace it with `buildinfo.Version`

Reply:
Fixed